### ISSUE TITLE
Entire Window Flashing

### ISSUE THREAD

**Issue by swindlesmccoop** [andmarti1424/sc-im#767]
I'm not sure how to describe it, so I just recorded a video: https://youtu.be/yufWiBUPYg8
I'm using DWM on X11 on OpenBSD. This is inside of a VirtualBox virtual machine as well. Not sure if the same error persists on real hardware, no current way to test.

**Comment by andmarti1424** [comment]
It probably has something to do with the X11 or ncurses on that particular system. Could you please test it outside X? And/or with another graphics card on the VM?

**Comment by idr4n** [comment]
I just installed sc-im manually on macOS Ventura 13.1 following the instructions and I am facing the same behavior. I didn't get any errors when installing though.

**Comment by swindlesmccoop** [comment]
> It probably has something to do with the X11 or ncurses on that particular system. Could you please test it outside X? And/or with another graphics card on the VM?

Yeah, I tested it outside of X and...it kinda just doesn't work. Not sure how to describe it but it should be in an entirely different issue. However, there was no flashing, so that's a plus.

**Comment by andmarti1424** [comment]
@swindlesmccoop what does not work?
if you dont get the flashing on X then it has something to do with your X, the WM or the virtual terminal..

**Comment by swindlesmccoop** [comment]
Yeah, I've resolved the flashing issue. However, the software itself simply does not seem to work properly. I haven't finished my testing but when I do, I'll make a new issue since it's entirely separate.

**Comment by renjub** [comment]
@swindlesmccoop, How did you resolve the flashing issue? I am facing same issue in CentOs

**Comment by swindlesmccoop** [comment]
I resolved the flashing issue by going to a TTY. It's some kind of terminal or WM issue that I haven't diagnosed yet.

**Comment by hymie0** [comment]
Just want to say that I am seeing the same issue on my freshly-updated copy sc-im in my X session.

So far I've determined that the "blink" commands in tui.c and file.c are **not** the problem.

The tags/0.8.1 version works correctly , but the tags/v0.8.2 version blinks.

That's where I'm stuck.

**Comment by Amooti73** [comment]
I have the same issue, but since I just wanted to convert a csv to an sc file, I could do it in spite of the flashing.  However, I proceeded to try sc-im on kitty and there was no flashing.  This is on openbsd 7.2 with dwm.

**Comment by bartenbach** [comment]
Seeing this exact same problem!

**Comment by cskeeters** [comment]
The wiki page [Building-SC-IM](https://github.com/andmarti1424/sc-im/wiki/Building-SC-IM) prompted macos users to install homebrew/dupes/ncurses, but if you do that, you get an error:

> Error: homebrew/dupes was deprecated. This tap is now empty and all its contents were either deleted or migrated.

I changed the command to be `brew install ncurses`.  It looks like this software includes `ncursesw`.  Not sure if the formula in dupes did something different.

I can confirm with this package,
1. the screen does flash in iTerm2, and
2. the background of the NORMAL style is white, even if it's set to black in `scimrc`.

If I use Alacritty, the text does not flash, but the background of normal cells is still white.
